### ISSUE TITLE
Bug: xxxx in 3d printing _wifi connection

### ISSUE THREAD

**Issue by harry-png** [Snapmaker/Luban#1391]
# 🐞 bug report

### Affected Version(s)

 The issue is caused by version 4.1.3

### Is this a regression? (optional)

 no, the previous version in which this bug was not present was: ?

### To Reproduce
 A clear and concise description of the problem...
After sending a file to snapmaker, you can start the printer from Luban. ( this ok). why if you go to the snapmaker display and want to control it further from there, the only way is to stop the print session, not a pause, only stopping the print job. this occurs also when lost the wifi connection.
why should it stop, the file is already sent to the printer snapmaker and the parameters from Luban also

Exception or Error (optional)
<pre><code>

<no->
</code></pre>

Expected behavior

- Operating System: macos Monterey
- Printer: [Snapmaker Original, Snapmaker A350/F350]
- Luban 4.1.3

</code></pre>

**Comment by paauserrat** [comment]
I agree, every time we send a file we need to disconnect the machine from wifi to be able to select the file and start the job. This needs to be fixed. we should be able to use the display when connected to wifi
